src/utils/functions.ts

```diff
@@ -230,48 +230,64 @@ const createConstructorInput = (contract: CompiledJSONOutput) => {
   writeConstructor(getConstructorInputFullPath(contract), contract, inputs);
 };
 
-const getGasEstimates = async (condition: string) => {
-  let estimate: EstimateGas;
-
-  axios
-    .get("https://api.blocknative.com/gasprices/blockprices", {
-      headers: {
-        "Access-Control-Allow-Origin": "*",
-        "Access-Control-Allow-Headers":
-          "Origin, X-Requested-With, Content-Type, Accept",
-      },
-    })
-    .then((res: any) => {
-      if (res.status === 200) {
-        switch (condition) {
-          case "Low": {
-            estimate = res.data.blockPrices[0].estimatedPrices.find(
-              (x: any) => x.confidence === 70
-            ) as EstimateGas;
-            break;
-          }
-          case "Medium": {
-            estimate = res.data.blockPrices[0].estimatedPrices.find(
-              (x: any) => x.confidence === 90
-            ) as EstimateGas;
-            break;
-          }
-          case "High": {
-            estimate = res.data.blockPrices[0].estimatedPrices.find(
-              (x: any) => x.confidence === 99
-            ) as EstimateGas;
-            break;
+const getNetworkBlockpriceUrl = (context: vscode.ExtensionContext) => {
+  const selectedNetwork = getSelectedNetwork(context);
+  if (selectedNetwork === "Polygon") {
+    return "https://api.blocknative.com/gasprices/blockprices?chainid=137";
+  } else if (selectedNetwork === "Ethereum") {
+    return "https://api.blocknative.com/gasprices/blockprices";
```

Comment:
`https://api.blocknative.com/gasprices/blockprices?chainid=1` this also returns mainnet blockprice. Use network id as a parameter.